XCUITests/ActivityStreamTest.swift

```diff
@@ -306,35 +306,36 @@ class ActivityStreamTest: BaseTestCase {
     func testActivityStreamPages() {
         let pagecontrolButton = TopSiteCellgroup.buttons["Next Page"]
         XCTAssertFalse(pagecontrolButton.exists, "The Page Control button must not exist. Only 5 elements should be on the page")
+        navigator.goto(BrowserTab)
```

Comment:
This line works because the fastest route to the loading a URL is chosen as `LoadURLByTyping`.

The problem is that the **Paste & Go** isn't available when there is no URL on the pasteboard before the menu is opened.

So: there are three places we can fix this:

1. as you have done. This will work, until you make a test that is alphabetically before this one.
2. Add the opening of the LongPressURLBar menu to `LoadByPasting`, and move [that action][1] to `BrowserTab`. 
3. Put the URL into the pasteboard right before performing the action, in [`Navigator.openURL()`][2]. 

(3) is definitely the easiest (it's one line).

[1]: https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FxScreenGraph.swift#L238-L241
[2]: https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FxScreenGraph.swift#L609